tests/benchmark/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM google/python
+
+RUN hg clone https://hg.python.org/benchmarks /app/benchmarks
```

Comment:
Ah, you're using the pybench benchmarks.  I would actually use the pypy benchmarks (https://bitbucket.org/pypy/benchmarks).  The unladen_swallow benchmarks, in particular, were developed at Google for testing real workloads.